src/providers/common.ts

```diff
@@ -161,6 +171,15 @@ export class Os {
     return os.name == this.name;
   }
 
+  /**
+  * Checks if the given OS is an Ubuntu Linux OS.
+  *
+  * @param os OS to compare
+  */
+  public isUbuntu() {
```

Comment:
All the other code I can see uses `os.isIn(_ALL_LINUX_UBUNTU_VERSIONS)`, so I would rather not include this method for consistency.